src/hera_pipelines/cli.py

```diff
@@ -0,0 +1,84 @@
+import sys
```

Comment:
Would like to get @aewallwi's opinion on whether moving his scripts to here will break any of the ongoing pipeline work. If they will, then I don't think it's worth it to remove stuff from utils.

Reply:
I agree